ratatool-examples/src/main/scala/com/spotify/ratatool/examples/scalacheck/GenExample.scala

```diff
@@ -151,4 +173,20 @@ object ExampleTableRowGen {
        */
       .tryAmend(intListGen)(_.getRecord("nullable_record").set("repeated_int_field"))
       .tryAmend(freqGen)(_.getRecord("nullable_record").set("frequency_string_field"))
+
+  private val recordIdGen: Gen[String] = Gen.alphaUpperStr
+
+  private val tableRowGenDup: Gen[TableRow] =
+    tableRowOf(tableSchema)
+
+  val exampleRecordGen2: Gen[(TableRow, TableRow)] =
```

Comment:
Can we name this something else. Maybe joinableExampleGen or something more descriptive. The name itself is up to you